network/nxos/nxos_switchport.py

```diff
@@ -69,6 +68,13 @@
         required: false
         default:  present
         choices: ['present','absent', 'unconfigured']
+    trunk_allowed_vlans:
+        description:
+            - if mode=trunk, these are the only VLANs that should be
```

Comment:
`If C(mode=trunk), these are the only VLANs that will be configured on the trunk.`

Assuming other VLANs will be removed.